src/components/behandlerdialog/meldingtilbehandler/MeldingTilBehandlerSkjema.tsx

```diff
@@ -131,7 +145,7 @@ export const MeldingTilBehandlerSkjema = () => {
             </Alert>
           )}
           <MeldingsType>
-            <SelectMeldingType />
+            <SelectMeldingType toggles={toggles} />
```

Comment:
Hvis `toggles` bare brukt brukt til å sende ned som en props her, kunne den da ikke bare leve i `SelectMeldingType` i stedet?

Reply:
Lurer på om jeg la det slik for at den kunne leve sammen med resten av staten. Tenkte også at det var større sjanse for at denne oftere måtte re-rendres, men dem tar jo ikke noe input, så høres ryddig ut 🙌